mu4e/mu4e-message.el

```diff
@@ -167,20 +168,26 @@ This is equivalent to:
 on PREFER-HTML and whether the message supports the given
 representation."
   (let* ((txt (mu4e-message-field msg :body-txt))
-	  (html (mu4e-message-field msg :body-html))
```

Comment:
Sorry about this. Any time I tried to edit any of the code near this function Emacs wanted to "fix" the indentation here... If the *new* indentation is right, I'm happy to submit it as a separate PR. If it is not, are you able to share what kind of settings you use to make Emacs indent it that way?